symphony-bdk-http/symphony-bdk-http-api/src/main/java/com/symphony/bdk/http/api/HttpClient.java

```diff
@@ -413,13 +417,16 @@ public HttpClient build() {
       return this.queryParams;
     }
 
-    public Map<String, Object> appendFormParam(String key, Object value) {
+    public Map<String, List<Object>> appendFormParam(String key, Object value) {
```

Comment:
aren't we breaking the API if we change that?
but then it looks like this part is internal only

Reply:
Yes this is internal only